spec/features/merchant/items/new_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+RSpec.describe "Merchant_items#new", type: :feature do
+  # US 11
```

Comment:
hey Kim! I'd recommend taking out the user story info here

Reply:
Shoot - I thought I'd caught them all.  I'll get this.